parcel-resolver-veda/index.d.ts

```diff
@@ -56,11 +56,23 @@ declare module 'veda' {
     MONTH = 'month',
     DAY = 'day'
   }
+  export interface LayerFrontmatterInfo {
+    info: {
+      description: string;
+      table: {
+        source: string;
+        spatialExtent: string;
+        latency: string;
+        unit: string;
```

Comment:
I guess this is the Sandbox thing, right? Could it basically just take whatever is in the config object and render that, rather than hard-coding keys?

Reply:
Yes it can. But that approach can be error-prone, and we do need to have at least these 4 to display at the layer level on the timeline if I am understanding the design right (unless we are fine with the data being irregular)